src/DependencyInjection/VichUploaderExtension.php

```diff
@@ -138,16 +140,22 @@ protected function registerAnnotationStrategy(ContainerBuilder $container, array
         }
 
         switch ($config['metadata']['type']) {
-            case 'attribute':
+            case 'annotation':
```

Comment:
This does not resolve the issue because this bundle's attribute reader requires `Doctrine\Common\Annotations\Reader` which is part of `doctrine/annotations`.

With this PR, when using attributes the following error is now thrown in src/Metadata/Driver/AttributeReader.php (line 15):

Attempted to load interface "Reader" from namespace "Doctrine\Common\Annotations".
Did you forget a "use" statement for "Symfony\Component\CssSelector\Parser\Reader"?


![Screenshot from 2023-02-06 12-35-20](https://user-images.githubusercontent.com/6660584/216972939-da3ef23a-15f5-48b0-aad0-52e884f116a5.png)

Reply:
It should be working now

Reply:
> This seems to be working for attributes +1

thank you for checking. Do you have the possibility to check it for annotations too? It would be awesome